astropy/wcs/wcs.py

```diff
@@ -2446,6 +2450,26 @@ def sip_foc2pix(self, *args):
         """.format(docstrings.TWO_OR_MORE_ARGS('2', 8),
                    docstrings.RETURNS('pixel coordinates', 8))
 
+    def proj_plane_pixel_scales(self):
+        """Same as :func:`astropy.wcs.utils.proj_plane_pixel_scales`
+        but returns a list of `~astropy.units.Quantity`.
+
+        """
+        from astropy.wcs import utils as wcs_utils  # Avoid circular import
+        val = wcs_utils.proj_plane_pixel_scales(self)
```

Comment:
Maybe use `values`? (More total nitpicking...)

Reply:
Done. I thought I was pretty progressive in using 3 chars, but apparently not progressive enough. 😸